Assets/HoloToolkit/Input/Scripts/GazeManager.cs

```diff
@@ -129,5 +129,17 @@ private void UpdateStabilizationPlane()
                 HolographicSettings.SetFocusPointForFrame(newFocusPointPosition, -gazeDirection);
             }
         }
+
+        /// <summary>
+        /// Sends message to the object currently focused on.
+        /// </summary>
+        /// <param name="message">Message to send</param>
+        public void SendMessageToFocusedObject(string message)
```

Comment:
Oh, yes, this relies on SendMessage being used to enable dynamically created objects to receive recognized keywords without creating additional infrastructure for registering strongly-typed callbacks at runtime.